recipe/build.sh

```diff
@@ -9,11 +9,30 @@ if [[ "${target_platform}" == osx-* ]]; then
   export CXXFLAGS="${CXXFLAGS} -DTARGET_OS_OSX=1"
 fi
 
+# For builds that are cross-compiled (aarch64), we need to build a bootstrap system first.
+# https://www.erlang.org/doc/installation_guide/install-cross#Build-and-Install-Procedure_Building-With-configuremake-Directly_Building-a-Bootstrap-System
+if [[ "${CONDA_BUILD_CROSS_COMPILATION}" -eq 1 ]]; then
+
+  BOOTSTRAP_PREFIX="${PREFIX}/../bootstrap"
```

Comment:
This should work

Another option we could consider is installing into `$BUILD_PREFIX` where all command line tools get installed. If we do that, we wouldn't need to mess with the `$PATH`. Also `$BUILD_PREFIX` is just tossed after the build. So it won't wind up in the package (unlike `$PREFIX`)

Reply:
Thanks, I'll use $BUILD_PREFIX for now.